app/src/ingestion/pdf_postprocess.py

```diff
@@ -114,8 +114,33 @@ def _add_link_markdown(e_text: EnrichedText) -> EnrichedText:
     return e_text
 
 
+def _add_list_markdown(
+    prev_e_text: EnrichedText | None, current_e_text: EnrichedText
+) -> EnrichedText:
+    if (
+        prev_e_text
+        and prev_e_text.type == TextType.LIST_ITEM
+        and current_e_text.type == TextType.LIST_ITEM
+    ):
+        # if sublist, indent 2 spaces
+        if "\u2022" in current_e_text.text:
+            current_e_text.text = "  - " + current_e_text.text
+            current_e_text.text = current_e_text.text.replace("\u2022", "\n  - ")
+            word_arr = current_e_text.text.split("\n")
+            filtered_dup_bullets = [
+                char for char in word_arr if char.strip() and char.strip() != "-"
+            ]
```

Comment:
Is this more correct naming:
```suggestion
            list_items = current_e_text.text.split("\n")
            nonempty_items = [
                item for item in list_items if item.strip() and item.strip() != "-"
            ]
```

Reply:
This logic seems very specific to the BEM pdfs but if it works (i.e., nested list items are correctly nested), then great! Basically the logic is: if there is a bullet character, then assume it's a nested list item one level down -- we don't handle deeper nesting.
I'd rerun the ingestion and check for nested items in the chunks to ensure they're correctly formatted.
Report any cases of nested list items that aren't identified as tech debt for a future PR.